### DISCUSSION TITLE
Apollo HD Map specification

### DISCUSSION THREAD

**Discussion by daohu527** [ApolloAuto/apollo#15117] [General]
Many people pay attention to the format of Apollo high-precision map, below is a 3.0 version

[EN_APOLLO_HDMap_OpenDrive_Specs_A3.0.1.pdf](https://github.com/ApolloAuto/apollo/files/12481150/EN_APOLLO_HDMap_OpenDrive_Specs_A3.0.1.pdf)

**Comment by haileynam** [comment]
I appreciate you sharing the information I needed. Thank you!
